storage/raft_transport.go

```diff
@@ -541,7 +538,9 @@ func (t *RaftTransport) SendAsync(req *RaftMessageRequest) bool {
 		// needs an explicit range ID.
 		panic("only heartbeat messages may be sent to range ID 0")
 	}
-	isSnap := req.Message.Type == raftpb.MsgSnap
+	if isSnap := req.Message.Type == raftpb.MsgSnap; isSnap {
+		panic("snapshots may not be sent asynchronously")
```

Comment:
Might be better to tell the reader what to do: `snapshots must be sent using SendSnapshot`.

`s/isSnap := req.Message.Type == raftpb.MsgSnap; isSnap/req.Message.Type == raftpb.MsgSnap/g`